music/src/piano_genie/model.ts

```diff
@@ -291,6 +361,75 @@ class PianoGenie {
     disposeState(this.lastState);
     this.initialized = false;
   }
+}
+
+/**
+ * Original Piano Genie config with autoregression and delta time features.
+ */
+class PianoGenieAutoDt extends PianoGenieBase {
+  // Execution state.
+  private lastOutput: number;
+  private lastTime: Date;
+  private time: Date;
+  private deltaTimeOverride: number;
+
+  // Inherits parent docstring.
```

Comment:
Yeah I don't think any of these "Inherits" comments are needed

Reply:
Wasn't sure about this. Thanks!